loans/src/main/java/com/microsevicesbank/loan/dto/ErrorResponseDTO.java

```diff
@@ -21,13 +21,12 @@ public class ErrorResponseDTO {
 
     @Schema(
             description = "Error code representing the error happened",
-            example = "500"
+            example = "XXX"
```

Comment:
If more error responses are included, it'll be necessary to update all these examples and make them much more precise.